src/rest/gmm_http_server.erl

```diff
@@ -13,7 +13,7 @@
 start_server() ->
     Dispatch = cowboy_router:compile([
         {'_', [
-            %%% @todo rest_vertices - mostly implemented
+            %%% @todo rest_vertices - implemented all except '/bulk'
```

Comment:
```suggestion
            %%% @todo implement '/bulk'
```

Reply:
Ok..
Mam już w sumie pomysł jak to poprawić, ogólnie chcę żeby ta długa lista była podzielona na mniejsze części z opisem który moduł to implementuje